packages/kotti-ui/source/kotti-filters/components/FilterRow.vue

```diff
@@ -104,11 +104,7 @@ export default defineComponent<{
 			type: String,
 		},
 		filter: {
-			default: (): KottiFilters.Filter => ({
```

Comment:
https://github.com/3YOURMIND/kotti/pull/689/files#diff-7447735e613d3621591bc6fc61f7aaa43f0feff8480db9da599fdb51af316461R8

you need a v-if on the `KtForm`

because now if the `filter` becomes a `null` prop -- you won't have the formKeys and the form will error